src/panels/lovelace/components/hui-card-options.ts

```diff
@@ -99,6 +99,7 @@ export class HuiCardOptions extends LitElement {
   static get styles(): CSSResult {
     return css`
       :host(:hover) {
+        overflow: hidden;
```

Comment:
This breaks the overflow menu

Reply:
![image](https://user-images.githubusercontent.com/5662298/93610239-1be7a800-f9cd-11ea-88d2-5f30bfe5936c.png)